integration/async-iterable-services-abort-signal/simple.ts

```diff
@@ -108,11 +108,12 @@ export interface Echoer {
   EchoBidiStream(request: AsyncIterable<EchoMsg>, abortSignal?: AbortSignal): AsyncIterable<EchoMsg>;
 }
 
+export const EchoerServiceID = "simple.Echoer";
```

Comment:
Kind of picky, but can we call these `EchoerServiceName` instead of `...ID`?

I'd remembered that services have names, i.e. from the `serviceDesc.name` field, and so initially thought the ID might be something else/different.

(At least I assume "name" is the most canonical term to use here--lmk if you think not.)

Reply:
Sure, that's fine with me. But I named it service id to be consistent with other places I'm using the value in starpc (RPC) which is mirroring drpc which I believe also uses the term service ID.

Either way is fine with me, I'll rename it if you don't comment otherwise

Reply:
Nice, I hadn't seen drpc before; I really like/agree with their rationale of "the out-of-the-box grpc runtime seems huge/overly complex".

I tried to do some more googling across the grpc docs/etc of "service id" vs. "service name", and am not finding an obvious winner, so yeah would appreciate going with name.

Thanks!